### ISSUE TITLE
Markdown in Atom Feed

### ISSUE THREAD

**Issue by davorg** [RapidApp/tpf-blog-live#11]
The [Atom feed](http://news.perlfoundation.org/atom.xml) contains unprocessed Markdown. This should converted to HTML before publishing the feed.

Currently, it looks really ugly on aggregators like [Perlsphere](http://perlsphere.net/).

**Comment by vanstyn** [comment]
The atom.xml was just a last minute hack to support something else, but now I'm glad I did, since it seems there are other consumers of it (fyi, the correct feed is now rss.xml, which is in the meta on the landing page). but, in any case, both are still working and now both properly render markdown into html...

But I just checked Perlsphere (i didn't know about it before you opened this issue) and it looks like they removed the tpf blog altogether? When you first opened the issue I saw what you were talking about, but now it seems gone. If you are affiliated with them, let them know we addressed the issue.

Thanks for the bug report
